### PR TITLE
Added a bunch of licenses

### PR DESCRIPTION
**Pull request by joellord**
Added a list of licenses as well as a license extractor script (from CSV files).
Also added some extra fields to the licenses JSON object like the OSI Approved field.  Those could be used in future versions of ianal.

### CODE REVIEW

**Comment by franciscop** [comment]
Hey thanks for the pull! I'll review it later today and accept it

**Comment by franciscop** [comment]
It gives mixed results.

Positive:

- `uri-path` is parsed as `?verify` instead of `this`, which is an improvement. 
- `tough-cookie`'s LICENSE shows `?verify`, which is an improvement on `The`. The README seems to be parsed incorrectly as `JSON License`, however it doesn't seem to be any standard license so both the original and this seem wrong in that part (so total score is an improvement).
- [`reduce-component`'s LICENSE and README](https://github.com/RedVentures/reduce) is parsed as `?verify` instead of the incorrect `Apache` (it's double licensed as MIT and Apache).

Negative:

- [`uglify-js`'s LICENSE](https://github.com/mishoo/UglifyJS2/blob/master/LICENSE) shows `?verify` where the original correctly showed `BSD`.
- [`wordwrap`'s LICENSE](https://github.com/substack/node-wordwrap/blob/master/LICENSE) (it shows `?verify`) where the original one recognizes it correctly as MIT.
- [`validate-npm-package-license`'s LICENSE](https://github.com/kemitchell/validate-npm-package-license.js/blob/master/LICENSE) shows `?verify`, where the original one recognizes it correctly as Apache.
- [`yargs`'s LICENSE](https://github.com/bcoe/yargs/blob/master/LICENSE) is recognized as `X11` instead of the original `MIT/X11` which is more clear.

Neutral:

- The name of the MIT is "MIT License" instead of just "MIT", and the same with others. While I liked more the concise type, it doesn't change the quality

Basically, the new method is too strict while the original method has more errors. I have checked it manually, but I'll try to get some tests going to verify it automatically. The ideal would be somewhere in the middle, I'll try to get something working as soon as possible. Thank you for the PR.

**Comment by franciscop** [comment]
@joellord I've retaken this package, I am not an expert in git and I'm not sure how to merge your changes now that there're many differences. I will just copy/paste and attribute you in the Readme if that's okay with you, thanks for the contribution

PS, this packages was called before IANAL but changed the name for the obvious (but not so obvious for me at the time) connotations.

**Comment by joellord** [comment]
Sure
